libs/execution_context/ebpf_protocol.h

```diff
@@ -104,6 +104,7 @@ typedef struct _ebpf_operation_create_map_request
 {
     struct _ebpf_operation_header header;
     struct _ebpf_map_definition ebpf_map_definition;
+    uint8_t data[1];
```

Comment:
suggest: probably would have been simpler to have a size field just above data[1] giving the length of the variable length buffer.

Reply:
that would also allow using the SAL `_Field_size_` [annotation](https://docs.microsoft.com/en-us/cpp/code-quality/annotating-structs-and-classes?view=msvc-160)

Reply:
Yes, we can add the size field. Currently though, for all the structures, the first variable length buffer does not have any size field.

Reply:
Ok to punt to a separate PR